cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/DatasetBasedStreamSizeScheduleStore.java

```diff
@@ -311,15 +334,33 @@ public void apply() throws Exception {
           if (txMethod != null) {
             txMethod.execute();
           }
-          byte[] rowKey = Bytes.toBytes(String.format("%s:%s", KEY_PREFIX,
-                                                      AbstractSchedulerService.scheduleIdFor(programId, programType,
-                                                                                             scheduleName)));
+          byte[] rowKey = Bytes.toBytes(getRowKey(programId, programType, scheduleName));
```

Comment:
PR referenced in @gokulavasan's comment - https://github.com/caskdata/cdap/pull/8504